linmod/tests/test_eval.py

```diff
@@ -0,0 +1,345 @@
+import numpy as np
+import polars as pl
+from numpy.random import default_rng
+from polars.testing import assert_frame_equal
+
+from linmod import eval
+from linmod.utils import expand_grid
+
+
+def _generate_fake_samples_and_data(
+    rng,
+    num_days,
+    num_divisions,
+    num_lineages,
+    num_samples,
+    sample_variance,
+):
+    r"""
+    Generate fake data $Y_{tgl}$ and "forecasts" $phi_{tgl}$.
+
+    - $Y_{tgl} \sim Uniform{0, ..., 99}$
```

Comment:
Is there a particular reason we're jumping through the hoop of discrete uniform -> normalize -> true proportion?

Reply:
I already have the infrastructure to convert counts into proportions (since we do that for data analysis too), so it was more immediately trivial to just drop in random counts as one column vector, instead of trying to generate probability vectors and ensure they are inserted into the data frame correctly

Not that that's much harder at all, so I don't mind changing if we want!

Reply:
That counts as a reason